ChartboostMediationDemo.xcodeproj/project.pbxproj

```diff
@@ -766,7 +722,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "#\"${PODS_ROOT}/Target Support Files/Pods-Demo-ChartboostMediationDemo-ObjC/Pods-Demo-ChartboostMediationDemo-ObjC-resources.sh\"\n";
+			shellScript = "\"${PODS_ROOT}/Target Support Files/Pods-Demo-ChartboostMediationDemo-ObjC/Pods-Demo-ChartboostMediationDemo-ObjC-resources.sh\"\n";
```

Comment:
This edit is introduced while using local source `pod 'ChartboostMediationSDK', :path => '..'` for development, and the following build error happens:
<img width="1925" alt="image" src="https://github.com/ChartBoost/chartboost-mediation-ios-sdk-demo/assets/122156786/99d15f46-2597-4995-90c7-61423364b86f">

This can be fixed by setting `ENABLE_USER_SCRIPT_SANDBOXING = NO;` as shown below. The UIKit and SwiftUI targets have this set to `NO` already, just the `ObjC` one is `YES`.
<img width="570" alt="image" src="https://github.com/ChartBoost/chartboost-mediation-ios-sdk-demo/assets/122156786/aed7a82d-d6ac-41eb-b40a-0f9d3fe75ec0">